### ISSUE TITLE
AgentMessage Component

### ISSUE THREAD

**Issue by spolu** [dust-tt/dust#1342]
Agent message component fully driven by the state coming from the Conversation component

It is composed of an Action component and a message component that should be aligned with the userMessage component

Some discussion to be done about the positioning of the picture since we might not always have messages and won't always have actions.

The Action component interface must be entirely generic (no retrieval word in the AgentMessage component).

Another task covers creating a RetrievalActionComponent which will be our first type of action component

**Comment by spolu** [comment]
depends on #1343

**Comment by lasryaric** [comment]
Leaving this opened as I am testing it right now.
